app/src/main/java/edu/uc/hornunjb/weatherwear/dto/Weather.kt

```diff
@@ -10,7 +10,11 @@ data class WeatherResponse(
     var wind: Wind,
     var sys: Sys,
     var name: String
-)
+){
+    override fun toString(): String{
+        return "It is ${weather[0].description} in ${name}"
+    }
+}
```

Comment:
Adding a toString() method to this class gives you a few options when actually processing the data. I'm not saying that the toString should display this exact data, but having it display something descriptive rather than just "object [Object]" is helpful for testing and usability purposes.